torch/testing/_internal/common_methods_invocations.py

```diff
@@ -7499,6 +7519,15 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
            skips=(
                SkipInfo('TestJit', 'test_variant_consistency_jit'),
            )),
+    OpInfo('pad',
+           dtypes=all_types_and_complex_and(torch.bool, torch.float16, torch.bfloat16),
+           supports_out=True,
+           supports_inplace_autograd=True,
+           test_neg_view=False,
```

Comment:
cc @anjali411 -- is there a comment that would clarify why this is False or some way to support it?

Reply:
I think I added this line by mistake. It passes if I remove this line